ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -224,48 +214,47 @@ objects:
             NOW="$(date +%s)"
           done
           [[ "${NOW}" -ge "${TARGET}" ]] && echo "timeout waiting for ${ROUTER_NAMESPACE}/${ROUTER_DEPLOYMENT} to be available" && exit 1
+
+          # wait until the image registry changes propogate to the apiserver to avoid
+          # unnecessary restarts
+          until oc get is -n openshift php 2>/dev/null; do
+            sleep 10
+          done
+          until [[ -n "$( oc get is -n openshift php --template '{{ .status.dockerImageRepository }}' 2>/dev/null )" ]]; do
+            sleep 10
+          done
+          # oh god the blood
+          sleep 180
         }
 
-        declare -fxr find_router_deployment
-        timeout 600 bash -ce 'find_router_deployment'
+        declare -fxr router_registry_check
+        timeout 600 bash -ce 'router_registry_check'
         EOF
-        cat  > "${HOME}"/run-tests.sh << 'EOF'
+        cat  > "${HOME}"/login.sh << 'EOF'
         #!/bin/bash
         set -euo pipefail
         export PATH=/home/packer:$PATH
 
-        function run-tests() {
-          mkdir -p /tmp/artifacts/junit/serial
-          if which openshift-tests && [[ -n "${TEST_SUITE-}" ]]; then
-            openshift-tests run "${TEST_SUITE}" --provider "${TEST_PROVIDER:-}" -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit
-            exit 0
-          fi
-          # TODO: remove everything after this point once we fork templates by release - starting with 4.0
-          if ! which extended.test; then
-            echo "must provide TEST_SUITE variable"
-            exit 1
-          fi
-          if [[ -n "${TEST_FOCUS:-}" ]]; then
-            ginkgo -v -noColor -nodes="${TEST_PARALLELISM:-30}" $( which extended.test ) -- \
-              -ginkgo.focus="${TEST_FOCUS}" -ginkgo.skip="${TEST_SKIP:-"\\[local\\]"}" \
-              -e2e-output-dir /tmp/artifacts -report-dir /tmp/artifacts/junit \
-              -test.timeout=2h ${PROVIDER_ARGS-} || rc=$?
-          fi
-          if [[ -n "${TEST_FOCUS_SERIAL:-}" ]]; then
-            ginkgo -v -noColor -nodes=1 $( which extended.test ) -- \
-              -ginkgo.focus="${TEST_FOCUS_SERIAL}" -ginkgo.skip="${TEST_SKIP_SERIAL:-"\\[local\\]"}" \
-              -e2e-output-dir /tmp/artifacts -report-dir /tmp/artifacts/junit/serial \
-              -test.timeout=2h ${PROVIDER_ARGS-} || rc=$?
-          fi
-          exit ${rc:-0}
+        # temporary - remove this check once a proper test is added to origin e2e
+        # and smoke-4 suite is running in libvirt
+        function kubeadmin_login() {
+        sleep 180
```

Comment:
The function body should gain a level of indentation.

Reply:
thanks, fixed.